doc/cascadia/profiles.schema.json

```diff
@@ -551,7 +551,9 @@
               "enum": [
                 "settingsFile",
                 "defaultsFile",
-                "allFiles"
+                "allFiles",
+                "settingsUI"
+                
```

Comment:
```suggestion
                "settingsUI"
```

Reply:
Also, IIRC the default is now "settingsUI" right? Mind fixing `"default"` and `"description"` too please.

Reply:
Done, let me know if the `"description"` is acceptable or if there's another way of describing it you'd prefer.